docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,69 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive needs a database to store the resources to make them retrievable.
+KubeArchive currently supports integration with PostgreSQL.
+MariaDB integration is in progress.
+
+[NOTE]
+====
+
+KubeArchive integration test suite runs with PostgreSQL 16.6
+
+====
```

Comment:
```suggestion
KubeArchive requires a database to store resources. This document list the databases supported, how to setup the database schema and how to configure KubeArchive to connect to the database.
```

Then I would create a section per DB: PostgreSQL, MariaDB. Each section should have the version, as you added in a NOTE below, and then the link to the SQL file.

What do you think?

Reply:
I've changed the docs to go with this approach, take a look and see if it looks good to you